resources/js/components/ui/Table/tquery_filters/IntFilterControl.tsx

```diff
@@ -88,7 +92,7 @@ export const IntFilterControl: VoidComponent<Props> = (props) => {
         <input
           name={`table_filter_from_${props.name}`}
           type="number"
-          class="h-full w-full border rounded"
+          class="h-full w-full border border-input-border rounded"
```

Comment:
nie znam tej klasy "border-input-border", to jakaś customowa?

Reply:
No, w tailwind.config.ts jest zdefiniowane input-border jako kolor.